tests/types.rs

```diff
@@ -129,10 +129,54 @@ fn hex_encoding() {
 
 #[test]
 #[cfg(feature = "serde")]
-fn test_key_with_big_array() {
+fn test_key_with_big_array_serde() {
     let rng = &mut StdRng::seed_from_u64(8586);
-    let s: Bytes64 = rng.gen();
-    let j = serde_json::to_string(&s).unwrap();
-    let s_back = serde_json::from_str(&j).unwrap();
-    assert!(s == s_back);
+    let bytes64: Bytes64 = rng.gen();
+    let bytes64_t = bincode::serialize(&bytes64).expect("Failed to serialize Bytes64");
```

Comment:
Seems `serde_json` is no longer a dependency and should be removed from `Cargo.toml`

Reply:
Done 👍